templates/form/fields.html.twig

```diff
@@ -10,17 +10,21 @@
 {% block date_time_picker_widget %}
     <div class="input-group date" data-toggle="datetimepicker">
         {{ block('datetime_widget') }}
-        <span class="input-group-addon">
-            <span class="fa fa-calendar" aria-hidden="true"></span>
-        </span>
+        <div class="input-group-append">
+            <button class="btn btn-outline-secondary">
```

Comment:
a button without an explicit `type` is a very bad idea: this makes it a submit button. and this is even worse here because this is indeed inside a form...

Reply:
So I replaced the `button` by a `span`

Reply:
using a button was the right thing to do from an accessibility point of view. But it needs an explicit `type=button` so that it is not a submit button.

Reply:
Ok I understand. Thanks.